content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx

```diff
@@ -43,12 +49,55 @@ model Profile {
 </tab>
 </TabbedContent>
 
-The `userId` relation scalar is a direct representation of the foreign key in the underlying database.This 1-1-relation expresses the following:
+The `userId` relation scalar is a direct representation of the foreign key in the underlying database. This one-to-one relation expresses the following:
 
 - "a user can have zero or one profiles" (because the `profile` field is [optional](/concepts/components/prisma-schema/data-model#type-modifiers) on `User`)
 - "a profile must always be connected to one user"
 
-</TopBlock>
+In this example, the `user` field of the `Profile` model references the `id` field of the `User` model. It is also possible to reference another field, as long as that field is marked with the `@unique` attribute. This is necessary to guarantee that there is only a single `User` connected to each `Profile`. In the following example, the `user` field references an `email` field in the `User` model, which is marked with the `@unique` attribute:
```

Comment:
```suggestion
In the previous example, the `user` relation field of the `Profile` model references the `id` field of the `User` model. It is also possible to reference another field, as long as that field is marked with the `@unique` attribute. This is necessary to guarantee that there is only a single `User` connected to each `Profile`. In the following example, the `user` field references an `email` field in the `User` model, which is marked with the `@unique` attribute:
```

Reply:
Consider making "It is also possible to" more active. E.g. "You can also"

For "as long as" consider to changing to "when" or "if". "As long as" might suggest a duration.

Reply:
Thanks, made both these changes